types/tcp-port-used/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
tslint json when adding new definitions cannot contain anything but extends clause. Please review rules clause.

Reply:
Thanks for the quick feedback! I'll fix this.